packages/cache/package.json

```diff
@@ -44,13 +44,13 @@
     "@actions/io": "^1.0.1",
     "@azure/abort-controller": "^1.1.0",
     "@azure/ms-rest-js": "^2.6.0",
-    "@azure/storage-blob": "^12.8.0",
+    "@azure/storage-blob": "^12.13.0",
     "semver": "^6.1.0",
     "uuid": "^3.3.3"
   },
   "devDependencies": {
     "@types/semver": "^6.0.0",
     "@types/uuid": "^3.4.5",
-    "typescript": "^3.8.3"
+    "typescript": "~4.8.0"
```

Comment:
The root level package.json uses a different `typescript` [version](https://github.com/actions/toolkit/blob/f3de1e53d628310756183415a00887f3ac6e15cd/package.json#L30). Wondering how Lerna resolves the version when running build/test commands.

Reply:
On first attempt I had received an error saying version mismatch, but running `npm install` and updating the `package-lock.json` file resolved the issue. I think it uses lockfile to resolve the dev dependencies as well.